www/scipylib/building/windows.rst

```diff
@@ -32,185 +32,257 @@ information than this document will provide.
+If the file isn't there, then poke around and try to find the file elsewhere. If you don't
+have that file, you'll probably need to find out what happened and then build OpenBLAS
+again. But if you have that file, we'll assume that you've completed this step correctly.
+Proceeding on that assumption, let's build SciPy.
+
+**Before continuing, it is extremely important that you remove every file named either**
+:code:`openblas.lib` **or** :code:`libopenblas.lib` **from your computer. Failure to do**
+**so could result in later build errors that will be difficult to debug. If you see the**
+**following output at any point, you have not completed this step correctly.**
+
+.. code:: shell
+
+   FOUND:
+      libraries = ['libopenblas', 'libopenblas']
+      library_dirs = ['C:\opt\64\lib']
+      language = c
+      define_macros = [('HAVE_CBLAS', None)]
+   FOUND:
+      libraries = ['libopenblas', 'libopenblas']
+      library_dirs = ['C:\opt\64\lib']
+      language = c
+      define_macros = [('HAVE_CBLAS', None)]
+
+Building SciPy
+==============
+
+Once you have built OpenBLAS, it's time to build SciPy. Before continuing make sure to
+install the following software for building on the latest Python version. For building
+on other Python versions, see the WindowsCompilers_ page.
+
+1) Install Microsoft Visual Studio 2015 or 2017 Community Edition (use the `build tools`_
+   from Microsoft)
+2) Finally, install Python from https://python.org/ (make sure to check the box to install
+   pip)
+
+After you've installed the required software, open an MSYS2 terminal, change to a good
+location to build, and clone SciPy.
+
+.. code:: shell
+
+   cd C:\Users\MyUser\Downloads
+   git clone https://github.com/scipy/scipy.git
+   cd scipy
+   
+Now we need to copy the :code:`openblas.a` file that we've built earlier to the correct
+location. If your Python is installed somewhere like the following:
+
+..code:: shell
+
+   C:\Users\<user name>\AppData\Local\Programs\Python\Python36\python.exe
+
+
+Then you'll need to put the :code:`openblas.a` file somewhere like the following:
+
+..code:: shell
+
+   C:\Users\<user name>\AppData\Local\Programs\Python\Python36\Lib
+
+Adjust the location accordingly based on where :code:`python.exe` is located. Now for a
+sanity check. Type  the following and press enter.
+
+.. code:: shell
+
+    gfortran
+
+If you see an error with the above command, :code:`gfortran` is not correctly installed.
+Go back to the "Building OpenBLAS" section and make sure that you have installed the correct
+tools.
+
+Now install the dependencies that we need to build and test SciPy. **It's important that you**
+**specify the full path to the native Python interpreter so that the built-in MSYS2 Python will**
+**not be used. Attempting to build with the MSYS2 Python will not work correctly.**
+
+**18 October 2017** A development NumPy is currently required to build. In addition, pip
```

Comment:
I would suggest mentioning the exact NumPy version via 

```
Until NumPy 1.14 is officially released, we have to work with the development version of the NumPy repository. You can still use the same OpenBLAS that you have built previously to also build NumPy. See the NumPy documentation....
```

Reply:
It would be good to point to a specific location in the numpy docs. But as far as I can see the numpy documentation does also need an overhaul with respect to building on Windows like we do here.

Reply:
I'm also working on more upstream changes so that we can have a CRT-compatible fortran compiler on Windows. The goal is to be able to run `vcpkg install openblas && pip install https://github.com/numpy/numpy.git`. But that's for another day.